recipe/meta.yaml

```
@@ -1,75 +1,70 @@
 {% set name = "mamba_gator" %}
 {% set version = "6.0.1" %}
-{% set build_number = 0 %}
 
 package:
-  name: {{ name|lower }}-meta
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 0bda2945e733d8633296bdc79a76487f44017c5b3089f61bf4098048bafe1b7b
 
 build:
-  noarch: python
-  number: {{ build_number }}
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
-outputs:
-  - name: {{ name|lower }}
-    build:
-      number: {{ build_number }}
-      noarch: python
-      entry_points:
-        - gator = mamba_gator.navigator.main:main
-    script: build_base.sh
-    requirements:
-      build:
-        - nodejs >=12.0.0,!=13,!=15,!=17,<18.0.0
-      host:
-        - python {{ python_min }}
-        - pip
-        - hatchling >=1.9.0
-        - hatch-jupyter-builder
-        - jupyterlab >=3.6.7,<5
-        - jupyter-packaging >=0.7.9,<0.8.0
-      run:
-        - python >={{ python_min }}
-        - jupyter_client
-        - jupyterlab_server >=2.0.0,<3.0.0
-        - packaging
-        - tornado
-        - traitlets
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - hatchling >=1.9.0
+    - hatch-jupyter-builder
+    - jupyterlab >=3.6.7,<5
+    - jupyter-packaging >=0.7.9,<0.8.0
```

https://github.com/mamba-org/gator/blob/v6.0.1/pyproject.toml#L5
```suggestion
    - jupyter-packaging >=0.7.9,<0.8.0
    - jupyter_server >=2.0.0,<3.0.0
```